### PR TITLE
note about placement of DAL before grappelli

### PR DESCRIPTION
**Pull request by chubz**

### CODE REVIEW

File: docs/install.rst [hunk 1]
```diff
@@ -23,10 +23,12 @@ This is to override the ``jquery.init.js`` script provided by the admin, which
 sets up jQuery with ``noConflict``, making jQuery available in
 ``django.jQuery`` only and not ``$``.
 
-Note: If you are trying to install from git, please make sure you are not using
+Notes:
+-If you are trying to install from git, please make sure you are not using
 **zip/archive** url of the repo ``django-autocomplete-light`` since it will not
 contain required submodules automatically. Otherwise these submodules will then
 need to be updated separately using ``git submodule update --init``.
+-If you are using django-autocomplete-light you need to also place ``dal``,``dal_select2`` ... above/before grappelli
```
Comment (jpic) [inline docs/install.rst]: Could you wordwrap please ?

With vim you can use gqq, with other text editor dunno ^^

Comment (jpic) [inline docs/install.rst]: Also, I think this should not be here, but rather replace:

> Then, let Django find static file we need by adding to :django:setting:`INSTALLED_APPS`, **before** django.contrib.admin:

With


> Then, let Django find static file we need by adding to :django:setting:`INSTALLED_APPS`, **before** django.contrib.admin **and** grappelli if present:

Comment (chubz) [inline docs/install.rst]: Sounds fine! Do you think that maybe then grappelli should be added above 'django.contrib.admin', in the installed app example also?
maybe just be commented out like this:
Then, let Django find static file we need by adding to
:django:setting:`INSTALLED_APPS`, **before** ``django.contrib.admin`` and
``grappelli`` if present::

    'dal',
    'dal_select2',
    # 'grappelli',
    'django.contrib.admin',

Comment (jpic) [inline docs/install.rst]: yes :guitar:

Comment (chubz) [inline docs/install.rst]: Added the second commit ... should be as we agreed now! Thanks for the comment!

Comment (jpic) [inline docs/install.rst]: Should it rather be "If you are using grappeli" ? Also, could you wrap words please ?

Thanks a lot !!

:guitar:

**Comment by jpic** [comment]
:guitar:

**Pull request merged**